src/components/Articles/Comments/Comments.jsx

```diff
@@ -0,0 +1,9 @@
+import SingleComment from "./SingleComment.jsx";
+
+export default function Comments({article, comments}){
+    return <section>
+        <h1 className={"title is-size-3 mb-6"}>Comments ({article.comment_count})</h1>
+        {comments.map(comment => <SingleComment comment={comment} key={comment.comment_id}/>)}
```

Comment:
At the moment, if our comments array is empty, we have nothing to feed this back to the user. Could you add some code in to account for this please?